RELEASES.md

```diff
@@ -72,6 +72,7 @@ deadline of 3 weeks.
 | 1.17.0  | 2020/12/31 | 2021/01/11 |  +11 days  | 2022/01/11  |
 | 1.18.0  | 2021/03/31 | 2021/04/15 |  +15 days  | 2022/04/15  |
 | 1.19.0  | 2021/06/30 | 2021/07/13 |  +13 days  | 2022/07/13  |
-| 1.20.0  | 2021/09/30 |            |            |             |
+| 1.20.0  | 2021/09/30 | 2021/10/05 |   +5 days  | 2022/10/13  |
+| 1.20.1  | 2021/12/30 |            |            |             |
```

Comment:
Curious to know why this is 1.20.1 and not 1.21.0 ?

Reply:
good spot @suniltheta - do you want to PR to amend ?